Better support for `en-us` locale (#1081)
handsontable/hyperformula: docs/guide/date-and-time-handling.md

# Date and time handling

The formats for the default date and time parsing functions can be set using configuration options:
- [`dateFormats`](../api/interfaces/configparams.md#dateformats),
- [`timeFormats`](../api/interfaces/configparams.md#timeformats),
- [`nullYear`](../api/interfaces/configparams.md#nullyear).

## Example

```javascript
const options = {
    // add popular date formats used in the USA

>     dateFormats: ['MM/DD/YYYY', 'MM/DD/YY', 'YYYY/MM/DD'],

Reviewer: Good question! I have started a thread on Slack. The proposed format for USA from the Cheat cheet (`M/d/yyyy`) does not even work when I apply it on your new demo: https://codesandbox.io/s/solitary-glade-reoy8w?file=/src/hyperformulaConfig.js

>     dateFormats: ['MM/DD/YYYY', 'MM/DD/YY', 'YYYY/MM/DD'],

Author: Removed

    // add a custom time format

>     timeFormats: ['hh:mm', 'hh:mm:ss', 'hh:mm:ss.sss'],

Author: Suggestions applied

# Date and time handling

The formats for the default date and time parsing functions can be set using configuration options:
- [`dateFormats`](../api/interfaces/configparams.md#dateformats),
- [`timeFormats`](../api/interfaces/configparams.md#timeformats),
- [`nullYear`](../api/interfaces/configparams.md#nullyear).

## Example

> The following date and time format configuration is used by default:

Reviewer: As suggested by you in another comment, maybe we should make it explicit that the default configuration is "European":

```suggestion
By default, HyperFormula uses European date and time format configuration:
```
